cherrypy/lib/httputil.py

```diff
@@ -42,7 +42,7 @@
 
 
 def urljoin(*atoms):
-    """Return the given path \*atoms, joined into a single URL.
+    r"""Return the given path \*atoms, joined into a single URL.
```

Comment:
I can't tell, but just like for the other change above, this change doesn't change the string content.

```
$ python3
Python 3.5.3 (default, May 10 2017, 15:05:55) 
[GCC 6.3.1 20161221 (Red Hat 6.3.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> """Return the given path \*atoms, joined into a single URL."""
'Return the given path \\*atoms, joined into a single URL.'
>>> r"""Return the given path \*atoms, joined into a single URL."""
'Return the given path \\*atoms, joined into a single URL.'
>>> 
```